packages/components/molecules/f-rating/src/components/Rating.vue

```diff
@@ -2,30 +2,97 @@
     <div
         :class="$style['c-rating']"
         data-test-id="rating-component">
-        {{ copy.text }}
+        <ul :class="$style['c-rating-star-wrapper']">
+            <li
+                v-for="star in maxStarRating"
+                :key="star"
+                :class="$style['c-rating-star']">
+                <star-filled-icon
+                    v-if="hasRating(star)"
+                    :class="$style['c-rating-star-filled']" />
+                <star-icon
```

Comment:
does this work for decimals? i.e. 3.5 stars being 3 filled and a half filled star etc

Reply:
Not yet. I have a note about a ticket for this. Also css was used before from what i remember and this is purely using svg no css.